core/templates/pages/contributor-dashboard-page/translation-opportunities/translation-opportunities.component.spec.ts

```diff
@@ -173,6 +173,21 @@ describe('Translation opportunities component', () => {
       });
   });
 
+  it('should move opportunities with no translatable cards to the bottom ' +
+    'of opportunity list', () => {
+    spyOn(translationLanguageService, 'getActiveLanguageCode').and.returnValue(
+      'en');
+    const {opportunitiesDicts, } = (
+      component.getPresentableOpportunitiesData({
+        opportunities: opportunitiesArray,
+        more: false
+      }));
+    expect(
+      opportunitiesDicts[opportunitiesDicts.length - 1].translationsCount +
+      opportunitiesDicts[opportunitiesDicts.length - 1].inReviewCount ===
+      opportunitiesDicts[opportunitiesDicts.length - 1].totalCount).toBeTrue();
```

Comment:
Can you check the content of the `opportunitiesDicts` before and after the function call?

Reply:
Thanks. Done.

Reply:
Can you add the check to this test too?

Reply:
Sure.

Reply:
Ah, sorry for not being clear. I meant to add a post-check to check the contents of `opportunitiesDicts`

Reply:
Oh, sorry about that.